### PR TITLE
Done

### PR DESCRIPTION
**Pull request by tsbutler**
Sorry I didn't comment this out.  I was just so happy to see it finally do a run without kicking up errors that I didn't want to jinx myself.

### CODE REVIEW

File: lib/reverse_string.rb [hunk 1]
```diff
@@ -1,5 +1,26 @@
 def reverse_characters(str)
+    i = 0
+    text = str
+    arr_1 = []
+    while i < text.length
+        i += 1
+        arr_1 << text[-i]
+        new_string = arr_1.join
```
Comment (sumeetjain) [inline lib/reverse_string.rb]: So `new_string` is being re-defined every time you pop a character from the string off. Is that necessary? How would your method function differently if you were to only do `new_string = arr_1.join` **once**, after the `while` loop had completely in its entirety?

**Comment by sumeetjain** [comment]
Nice! This is approvable, but I've asked a question about your code that I'd like you to consider first.

**Comment by tsbutler** [comment]
I actually found that I could shift the new_string = arr_1.join line to after the end of the loop and so not repeat that particular step needlessly.  Also, I could completely forgo the use of the "return new_string".

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**